package.json

```diff
@@ -1,63 +1,63 @@
 {
-    "name": "mars-mission-frontend",
-    "version": "0.1.0",
```

Comment:
Why did this file change?

Reply:
im not sure as the only thing i added was z-index: 1; to the scss file?

i had issues with getting the latest version of the master so i deleted the mars-mission-frontend folder from my laptop and cloned it again and ran npm install, could this be the reason?